dc.external.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+    reader:
+        build:
+            context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+            dockerfile: Dockerfile.prod
+            args:
+                MODULE: "reader"
+                PORT: "9010"
+        image: openslides-datastore-reader
+        ports:
+            - "9010:9010"
+        depends_on:
+            - postgresql
+        networks:
+            - postgresql
+    writer:
+        build:
+            context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+            dockerfile: Dockerfile.prod
+            args:
+                MODULE: "writer"
+                PORT: "9011"
+        image: openslides-datastore-writer
+        ports:
+            - "9011:9011"
+        environment:
+            - DATASTORE_ENABLE_DEV_ENVIRONMENT=1
+            - COMMAND=create_initial_data
+            - DATASTORE_INITIAL_DATA_FILE=https://raw.githubusercontent.com/OpenSlides/OpenSlides/openslides4-dev/docs/example-data.json
+        depends_on:
+            - postgresql
+            - redis
+        networks:
+            - postgresql
+            - redis
+    postgresql:
+        image: postgres:11
+        env_file: database.env
```

Comment:
For the portability: You can move the three env variables into this file. It will become longer, but then it's just this file to copy.